tests/test_server.py

```diff
@@ -93,3 +93,8 @@ def test_get_all_payments(self):
         resp = self.app.get('/payments')
         self.assertEqual(resp.status_code, status.HTTP_200_OK)
         self.assertEqual(len(json.loads(resp.data)), 2)
+
+    def test_list_all_payment_methods(self):
+        """Get all payment methods"""
+        resp = self.app.get('/payments/methods')
+        self.assertEqual(resp.status_code, status.HTTP_200_OK)
```

Comment:
This should assert that the list of payment methods matches the values in the enum instead of just looking at the response code.

Reply:
updated the test case to check the length of the responses and a test to check if the values are equal.

Reply:
Updated the test case to avoid hardcoding and using a Set instead of a List as discussed.